### PR TITLE
adds hash to response of wallet/postTransaction

### PR DESCRIPTION
**Pull request by hughy**
## Summary

sendTransaction includes the hash in its responses, but postTransaction does not.

a client _could_ hash the output themselves, but this adds an extra step and perhaps an extra dependency for the client.

## Testing Plan

- updates unit tests

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[X] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish/src/rpc/routes/wallet/postTransaction.test.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ describe('Route wallet/postTransaction', () => {
 
     expect(addSpy).toHaveBeenCalledTimes(0)
     expect(response.status).toBe(200)
+    expect(response.content.hash).toBeDefined()
```
Comment (EvanJRichard) [inline ironfish/src/rpc/routes/wallet/postTransaction.test.ts:30]: Is it worth it to calculate what the `hash` should be and check equality, do you think?

**Review by EvanJRichard — APPROVED** [review]
Simple, convenient, and unifies RPCs a little more. One question about tests, but non-blockin g. :shipit:

File: ironfish/src/rpc/routes/wallet/postTransaction.test.ts (hunk 1 above)
Comment (hughy) [reply ironfish/src/rpc/routes/wallet/postTransaction.test.ts:30]: It's a little tricky to do this: there's randomness involved in posting transactions, so posting the same raw transaction twice will result in two different hashes 😓

Comment (EvanJRichard) [reply ironfish/src/rpc/routes/wallet/postTransaction.test.ts:30]: Ah, true. Forget it, then, imo.

**Review by danield9tqh — APPROVED** [review]

**Review by NullSoldier — APPROVED** [review]

File: ironfish/src/rpc/routes/wallet/postTransaction.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (rohanjadvani) [reply ironfish/src/rpc/routes/wallet/postTransaction.test.ts]: I don't think this works in Jest with `toBe`. It seems like CI is failing with this.

**Pull request merged**